docs/components/console/manage-clusters/manage-api-clients.md

```diff
@@ -8,6 +8,8 @@ To interact with Zeebe in the cloud from the outside, every client application m
 
 ![auth-flow](./img/client-auth.png)
 
+The application authenticates itself with Auth service using Client Id and Client Secret, Auth service validates this information and returns an access token, and the application can then use this access token to interact with Zeebe in the cloud.
```

Comment:
```suggestion
The application authenticates itself with Auth service using `Client Id` and `Client Secret`, Auth service validates this information and returns an access token, and the application can then use this access token to interact with Zeebe in the cloud.
```
Should "Auth" be lowercase here?

Reply:
I wonder if this should actually be OAuth throughout, or am I misunderstanding?

Reply:
Oauth makes sense!

Reply:
Resolved in latest commit🤠